mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -766,8 +766,7 @@ func TestAccComposerEnvironment_composerV2MasterAuthNetworksUpdate(t *testing.T)
 	})
 }
 
-// Checks behavior of node config, including dependencies on Compute resources.
-func TestAccComposerEnvironment_withNodeConfig(t *testing.T) {
+func TestAccComposer1Environment_withNodeConfig_withTags(t *testing.T) {
```

Comment:
Maybe remove `_withTags` suffix here, since the test checks entire node_config, not only Tags

Reply:
Updated.